File: app/src/main/java/com/apps/LauncherPlaceholderFragment.java
package com.apps;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.yuki.yukihub.databinding.FragmentLauncherPlaceholderBinding;

public class LauncherPlaceholderFragment extends Fragment {
    private static final String ARG_TITLE = "title";

    private FragmentLauncherPlaceholderBinding binding;

    public static LauncherPlaceholderFragment newInstance(String title) {
        LauncherPlaceholderFragment fragment = new LauncherPlaceholderFragment();
        Bundle args = new Bundle();
        args.putString(ARG_TITLE, title);
        fragment.setArguments(args);
        return fragment;
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        binding = FragmentLauncherPlaceholderBinding.inflate(inflater, container, false);
        return binding.getRoot();
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        Bundle args = getArguments();
        binding.tvPlaceholderTitle.setText(args == null ? "占位" : args.getString(ARG_TITLE, "占位"));
    }

    @Override
    public void onDestroyView() {
        super.onDestroyView();
        binding = null;
    }
}

File: app/src/main/java/com/apps/home/LauncherPlaceholderFragment.kt
package com.apps.home

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.apps.widget.LauncherTabletPortraitScaler
import com.yuki.yukihub.databinding.FragmentLauncherPlaceholderBinding

class LauncherPlaceholderFragment : Fragment() {
    private var binding: FragmentLauncherPlaceholderBinding? = null

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        return FragmentLauncherPlaceholderBinding.inflate(inflater, container, false)
            .also { binding = it }
            .root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        val currentBinding = requireNotNull(binding)
        LauncherTabletPortraitScaler.apply(currentBinding.root)
        currentBinding.tvPlaceholderTitle.text = arguments?.getString(ARG_TITLE, "占位") ?: "占位"
    }

    override fun onDestroyView() {
        binding = null
        super.onDestroyView()
    }

    companion object {
        private const val ARG_TITLE = "title"

        @JvmStatic
        fun newInstance(title: String?): LauncherPlaceholderFragment = LauncherPlaceholderFragment().apply {
            arguments = Bundle().apply { putString(ARG_TITLE, title) }
        }
    }
}
